src/pages/Search/SearchBooks.js

```diff
@@ -0,0 +1,139 @@
+import React, { useState } from 'react';
+import '../../styles/theme';
+import styled from 'styled-components';
+
+const filters = [
+  { filter: '전체', placeholder: '검색어를 입력하세요' },
+  { filter: '저자', placeholder: '저자를 입력하세요' },
+  { filter: '제목', placeholder: '제목을 입력하세요' },
+  { filter: '출판사', placeholder: '출판사를 입력하세요' },
+];
+
+export default function SearchBooks() {
+  const [filter, setFilter] = useState('전체');
+  const [visible, setVisible] = useState(false);
+  const [placeholder, setPlaceholder] = useState('검색어를 입력하세요.');
+  const [searchValue, setSerchValue] = useState('');
+
+  const handleFilter = (filter, placeholder) => {
+    setFilter(filter);
+    setPlaceholder(placeholder);
+    setVisible(false);
+  };
+
+  const onChangeSearchValue = e => {
+    setSerchValue(e.target.value);
+  };
+
+  const handleCleanSearch = e => {
+    setSerchValue('');
+  };
```

Comment:
```suggestion
  const handleSearchValue = e => {
    setSerchValue(e.target.value);
  };

  const clearSearchValue = () => {
    setSerchValue('');
  };
```
- 함수 명명하는 방법에 좀 더 일관성이 있으면 읽기 좋을 것 같습니다
- 사용하지 않는 인자는 제거해주세요

Reply:
수정 완료했습니다